sagemaker_train/cfg_sample.yaml

```diff
@@ -0,0 +1,37 @@
+accum-freq: 4
+beta1: 0.9
+beta2: 0.95
+data-key: "json"
+dataset-resampled: ""
```

Comment:
If you set this to True instead of empty string, the error you mentioned should go away when passing the config via path (and similarly for all other keys which have a "" value - set them to True instead)

Reply:
I tried that and it still gives the same error. It seems to read everything as a string.

Just to double-check: The way to pass a config is to just do `train_args = {"config": args.cfg_path}` instead of the yaml.safe_load(f), right?

Reply:
Hmm yeah that should be all you need. And you rebuilt the docker container after that change right? Will try it out later today, maybe there's something wrong with the parsing logic.